interface/main/finder/dynamic_finder.php

```diff
@@ -1,5 +1,5 @@
 <?php
-// Copyright (C) 2012 Rod Roark <rod@sunsetsystems.com>
+// Copyright (C) 2012, 2016 Rod Roark <rod@sunsetsystems.com>
```

Comment:
recommend not changing the copyright lines of somebody else. instead better to add your own copyright line with your name/email.